smg-automotive/components-pkg
# Extract gallery components

[IN-942](https://autoricardo.atlassian.net/browse/IN-942)

## Motivation and context

GalleryHeader component can be shared across listings and seller

## Before

We had this component in both projects

## After

It is now extracted to components-pkg: https://extract-gallery-components-components-pkg.branch.autoscout24.dev/?path=/story/components-navigation-galleryheader--gallery-header

## How to test

It should behave the same as before in both projects, see [this PR](https://github.com/smg-automotive/listings-web/pull/870) and [this one](https://github.com/smg-automotive/seller-web/pull/530)


[IN-942]: https://autoricardo.atlassian.net/browse/IN-942?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
